frontend/src/types.ts

```diff
@@ -1308,6 +1310,13 @@ export interface FlattenedFunnelStepByBreakdown {
     significant?: boolean
 }
 
+export enum BreakdownAttributionType {
+    FirstTouch = 'first_touch',
+    LastTouch = 'last_touch',
+    AnyStep = 'all_events',
```

Comment:
any step for unordered is same as step = 0. 

All_events is for the 4th option, where we want the property to be present on all events.

Reply:
so you want it to be all 4 options available? So not like the mocks where it's only 3 at all times then? https://www.figma.com/file/gQBj9YnNgD8YW4nBwCVLZf/PostHog-App?node-id=11313%3A49272

Reply:
I can update it but if it's also faster for you to make these tweaks yourself feel free to! Either works for me :)